chef-ha-cluster/README.md

```diff
@@ -7,23 +7,21 @@
 <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-**This template has artifacts that need to be staged for deployment (Configuration Scripts) so use the below command with the upload flag to deploy this template or provide a storage account and SAS token when using the deploy button above.**
-You can optionally specify a storage account to use, if so the storage account must already exist within the subscription.  If you don't want to specify a storage account
-one will be created by the script (think of this as "temp" storage for AzureRM) and reused by subsequent deployments.
-
-```PowerShell
-.\Deploy-AzureResourceGroup.ps1 -ResourceGroupLocation 'eastus' -ArtifactsStagingDirectory 'chef-ha-cluster' -UploadArtifacts 
-```
-```bash
-azure-group-deploy.sh -a chef-ha-cluster -l eastus -u
-```
-
-This template deploys a Chef Backend High-Availability Cluster.
+This template deploys a Chef High-Availability Cluster.
 `Tags: chef,cluster,ha`
 
 ## Deployment steps
 
-You can click the "deploy to Azure" button at the beginning of this document or follow the instructions for command line deployment using the scripts in the root of this repo.
+1. Provision a Standard (LRS) storage account, or use an existing one (must be Standard)
```

Comment:
Probably should leave the instructions about how to do it with a script but I don't mind adding the instructions about how to create one manually in the UI / do it from the portal.  The updates to reference the scripts from github should work fine as well -- the SASToken is used by the scripts to write back into the storage account but the scripts themselves can live wherever.